src/styles/index.scss

```diff
@@ -2,2 +1,134 @@
+@import './variables';
+
+.page-body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    background-image: url(../images/imac.jpeg);
+    background-size: 160px 134px;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    width: 100%;
+    color: $main-color;
+    font-size: 12px;
+    font-weight: 500;
+    font-family: inherit;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    font-family: inherit;
```

Comment:
The same applies here for the `font-family` property. It's not necessary to declare it again.